blackbox/docs/admin/system-information.rst

```diff
@@ -1114,26 +1114,26 @@ currently being executed in the cluster.
 Table Schema
 ............
 
-+------------------------------------+----------------------------------------------------+---------------+
-| Column Name                        | Description                                        |  Return Type  |
-+====================================+====================================================+===============+
-| ``id``                             | The job UUID.                                      | ``TEXT``      |
-|                                    |                                                    |               |
-|                                    | This job ID is generated by the sytem.             |               |
-+------------------------------------+----------------------------------------------------+---------------+
-| ``node``                           | Information about the node that created the job.   | ``OBJECT``    |
-+------------------------------------+----------------------------------------------------+---------------+
-| ``node['id']``                     | The id of the node.                                | ``TEXT``      |
-+------------------------------------+----------------------------------------------------+---------------+
-| ``node['name']``                   | The name of the node.                              | ``TEXT``      |
-+------------------------------------+----------------------------------------------------+---------------+
-| ``started``                        | The point in time when the job started.            | ``TIMESTAMP`` |
-+------------------------------------+----------------------------------------------------+---------------+
-| ``stmt``                           | Shows the data query or manipulation statement     | ``TEXT``      |
-|                                    | represented by this job.                           |               |
-+------------------------------------+----------------------------------------------------+---------------+
-| ``username``                       | The user who is executing the statement.           | ``TEXT``      |
-+------------------------------------+----------------------------------------------------+---------------+
++------------------------------------+----------------------------------------------------+------------------------------+
```

Comment:
Minor nitpick: Maybe the left column could be made a bit narrower so that the table is better readable within the github diff view?